code/game/gamemodes/cult/cult.dm

```diff
@@ -59,8 +82,8 @@
 		restricted_jobs += "Assistant"
 
 	//cult scaling goes here
-	recommended_enemies = 1 + round(num_players()/8)
-	orbs_needed = recommended_enemies
+	recommended_enemies = 3 + round(num_players()/20)
```

Comment:
>scaling starting cultists when their entire objective is to have X people and the round ends

At least it's a bit more sane than the previous oldcult scaling that would routinely give you 8-9 people I guess